src/style.css

```diff
@@ -0,0 +1,95 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 11px;
+}
+
+body,
+p {
+  margin: 0;
+  padding: 0;
+}
+
+.main__card {
+  width: 200px;
+  padding: 14px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product-image {
+  height: 134px;
+  width: 160px;
+  margin: 16px 5px 45px;
+  display: flex;
+  justify-content: center;
+}
+
+.main__product-name {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 15px;
+  margin-bottom: 5px;
+}
+
+.main__product-code {
+  margin-bottom: 15px;
+}
+
+.main__rating-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 20px;
+}
+
+.main__product-stars {
+  display: flex;
+  justify-content: space-between;
+  flex-direction: row;
```

Comment:
redundant

Reply:
it is redundant because  flex-direction: row is default